spec/spec_helper.rb

```diff
@@ -1,3 +1,5 @@
+require "capybara/rspec"
```

Comment:
This does not belong here. Not sure if you need it. But if you do, a suggestion to move to `spec/support/capybara_webkit.rb`?

Reply:
Actually I don't, spec pass also without. Deleted, thanks!